README.md

```diff
@@ -1,18 +1,17 @@
 # FaunaDB Go Driver
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/fauna/faunadb-go)](https://goreportcard.com/report/github.com/fauna/faunadb-go)
-[![GoDoc](https://godoc.org/github.com/fauna/faunadb-go/faunadb?status.svg)](https://pkg.go.dev/github.com/fauna/faunadb-go/v3)
+[![GoDoc](https://godoc.org/github.com/fauna/faunadb-go/faunadb?status.svg)](https://pkg.go.dev/github.com/fauna/faunadb-go/v4)
 [![License](https://img.shields.io/badge/license-MPL_2.0-blue.svg?maxAge=2592000)](https://raw.githubusercontent.com/fauna/faunadb-go/master/LICENSE)
 
 A Go lang driver for [FaunaDB](https://fauna.com/).
 
 ## Supported Go Versions
 
 Currently, the driver is tested on:
-- 1.11
-- 1.12
 - 1.13
 - 1.14
+- 1.15
```

Comment:
Do we want to include 1.16 here? I've been running my doc example tests for the new third-party auth functions on 1.16.

Reply:
Done.